ubuntudesign/ubuntu-vanilla-theme
# Add tick lists

#Details
 Card: https://canonical.leankit.com/Boards/View/111185042/116255901

##Done 
Add list tick example to demo.

#QA
Update your shizzle then head to demo/index.html and check that list-ticks (or a more suitable name?) is present and that it uses the brand color as the bullet background color. You can make sure this works by changing the brand color in _global-settings.scss then run gulp build or gulp test.
